ConditionalExpression/Conditional_Statements.js

```diff
@@ -0,0 +1,34 @@
+// if Statement
+let marks = 84;
+if (marks > 60) {
+  console.log("You are Pass");
+}
+
+// else Statement
+let date = 19;
+if (date == 18) {
+  console.log("Today Date Is : " + `${date}`);
```

Comment:
can be written as 
console.log(`Today Date is : ${date}`);
use of template String